src/widget/wtrackproperty.h

```diff
@@ -37,9 +43,11 @@ class WTrackProperty : public WLabel, public TrackDropTarget {
     void updateLabel();
 
     const char* m_pGroup;
-    UserSettingsPointer m_pConfig;
+    const UserSettingsPointer m_pConfig;
     TrackPointer m_pCurrentTrack;
     QString m_property;
+
+    WTrackMenu *m_pMenu;
```

Comment:
`const parented_ptr<WTrackMenu>` because it could be initialized in the initializer list of the constructor

Reply:
We cannot initialize `m_pMenu` in the initializer list because `m_pMenu` will require `trackModel` which is not loaded until the slot `WTrackTableView::loadTrackModel` is called. This explains the placement of `m_pMenu`'s initialization in the `loadTrackModel` function.

Reply:
That's correct for WTrackTableView, but we can put it in the constructor for WTrackProperty.